modules/virt-using-hostpath-provisioner.adoc

```diff
@@ -15,6 +15,11 @@ To deploy the hostpath provisioner and enable your virtual machines to use local
 ====
 The backing directory must not be located in the filesystem's root directory because the `/` partition is read-only on {op-system-first}. For example, you can use `/var/<directory_name>` but not `/<directory_name>`.
 ====
++
+[NOTE]
+====
+If you select a directory that shares space with your operating system, you might exhaust the space on that partition and your node becomes non-functional. Create a separate partition and point the hostpath provisioner to the separate partition to avoid interference with your operating system.
```

Comment:
I would use the same language as used in the upstream document. This has serious consequences and should be a warning:

"WARNING If you select a directory that shares space with your Operating System, you can potentially exhaust the space on that partition and your node will become non-functional. It is recommended you create a separate partition and point the hostpath provisioner there so it will not interfere with your Operating System."

Reply:
@kgoldbla Change the Note to a Warning.